CMakeLists.txt

```diff
@@ -38,6 +39,12 @@ include(Builtins)
 
 add_library(VecCore INTERFACE)
 
+set(LIBMVEC check_library_exists(m _ZGVcN4v_sin "" LIBMVEC))
+if (LIBMVEC)
+  set( CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Ofast -fopenmp" )
```

Comment:
Should this be done in another way ? Is this the reason why it doesn't work for clang ?

Reply:
Yes, not all compilers use only `-fopenmp` to compile. Please have a look at https://cmake.org/cmake/help/v3.11/module/FindOpenMP.html, that should help you setup OpenMP in a more general way that will work with all compilers.